<?xml version="1.0" ?>
<launch>
  <!--
      Launch file for a CPR closed-loop controller.
  -->

  <!-- Connection params-->
  <arg name="IP_ADDRESS_ARM" default="localhost"/>
  <arg name="REVERSE_PORT_ARM" default="50001"/>
  <arg name="CMD_TOPIC_PLATFORM" default="/cmd_vel"/>


  <param name="ip_address_arm" value="$(arg IP_ADDRESS_ARM)"/>
  <param name="reverse_port_arm" value="$(arg REVERSE_PORT_ARM)"/>
  <param name="cmd_topic_platform" value="$(arg CMD_TOPIC_PLATFORM)"/>

  <node pkg="cpr_closed_loop_controller" name="cpr_closed_loop_controller"
  type="cpr_closed_loop_controller_node" output="screen"/>

</launch>
